gatorgrouper/utils/constants.py

```diff
@@ -4,9 +4,9 @@
 ALGORITHM_ROUND_ROBIN = "rrobin"
 ALGORITHM_RANDOM = "random"
 DEFAULT_METHOD = "random"
-DEFAULT_CSVFILE = "students.csv"
+DEFAULT_CSVFILE = "/home/w/wuj/cs203S2019/lab/students.csv"
```

Comment:
This should definitely ***not*** be the default.

Reply:
Yes. I just put it there when I was trying to fix the path. It is not even using it right now.

Reply:
It should not be committed, and be removed, because this kind of path changing can make it really easy for weird errors to happen to future programmers.

Reply:
Will do that, I think just forgot to discard that change when I push.